### PR TITLE
[Merged by Bors] - feat: compute the exponent of `ZMod`

### PR DESCRIPTION
**Pull request by j-loreaux**
---

I'm not sure if this is the right file to place this, but I didn't see another one, and certainly this is the earliest available file.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by alexjbest** [comment]
LGTM

maintainer merge

File: Mathlib/GroupTheory/SpecificGroups/Cyclic.lean [hunk 1]
```diff
@@ -602,4 +602,12 @@ theorem IsCyclic.exponent_eq_zero_of_infinite [Group α] [IsCyclic α] [Infinite
 #align is_cyclic.exponent_eq_zero_of_infinite IsCyclic.exponent_eq_zero_of_infinite
 #align is_add_cyclic.exponent_eq_zero_of_infinite IsAddCyclic.exponent_eq_zero_of_infinite
 
+instance ZMod.instIsAddCyclic (n : ℕ) : IsAddCyclic (ZMod n) where
+  exists_generator := ⟨1, fun n ↦ ⟨n, by simp⟩⟩
+
+protected theorem ZMod.exponent (n : ℕ) : AddMonoid.exponent (ZMod n) = n := by
```
Comment (eric-wieser) [inline Mathlib/GroupTheory/SpecificGroups/Cyclic.lean:608]:
```suggestion
@[simp]
protected theorem ZMod.exponent (n : ℕ) : AddMonoid.exponent (ZMod n) = n := by
```

**Review by eric-wieser — APPROVED** [review]
bors d+

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**